_posts/2023-04-10-opensource-perf.markdown

```diff
@@ -0,0 +1,127 @@
+---
+layout: post
+title:  "Improving JSON parsing performance in opensearch-java"
+authors:
+  - chibenwa
+date:   2023-04-10
+categories:
+  - technical-post
+meta_keywords: OpenSearch, Apache James, Flame graph, Performance, Driver
+meta_description: Learn how contributors of Apache James found out and fixed major performance problems in OpenSearch Java driver JSON parsing.
+---
+
+As an open source enthusiast, I believe in the power of collaboration to make open source projects faster and more efficient. In this blog post, I will share how my team at [LINAGORA](https://linagora.com), contributing to the Apache James project, identified and addressed a performance issue in OpenSearch using benchmark tools and flamegraphs, and collaborated with the OpenSearch community to make the project better.
```

Comment:
I would remove "to make the project better" because it dilutes the "identified and addressed a performance issue"